libs/designsystem/src/lib/components/item/item.component.scss

```diff
@@ -101,18 +101,10 @@
 /* clean-css ignore:start */
 // Fixes https://github.com/kirbydesign/designsystem/issues/1745
 ion-item {
-  :host-context(kirby-list-item:first-of-type) & {
-    --padding-top: #{utils.size('xxs')};
-  }
-
   :host-context(kirby-list-item:first-of-type kirby-card) & {
     --padding-top: 0;
   }
```

Comment:
Could it be that the two other rules be obsolete now? They seem to reset the additional padding in a specific scenario, but now there is no additional padding.

Reply:
Yeah that could be. I'm gonna do a quick test to make sure we don't break anything :)

Reply:
You're right. We can remove it! :)